morphic.js

```diff
@@ -4496,6 +4496,56 @@ CursorMorph.prototype.init = function (aStringOrTextMorph) {
         this.target.setAlignmentToLeft();
     }
     this.gotoSlot(this.slot);
+    
+    // Add hidden text box for copying and pasting
+    this.hiddenText = document.createElement('textarea');
+    this.target.hiddenText = this.hiddenText;
```

Comment:
Ah, I already thought so but wasn't sure whether or not there might be some other functionality depending on this. Thanks, @cyderize, for pioneering this wonderful and badly needed feature!